src/requests/hover.jl

```diff
@@ -46,7 +46,7 @@ function get_hover(b::StaticLint.Binding, documentation::String, server)
                 end
             end
         else
-            try
+            documentation = try
```

Comment:
Is that needed? Aren't all code paths already assigning things to `documentation`?